package/src/contexts/messageInputContext/hooks/useMessageDetailsForState.ts

```diff
@@ -58,6 +53,16 @@ export const useMessageDetailsForState = <
     }
   }, [text]);
 
+  useEffect(() => {
+    if (
+      !isEditingBoolean<At, Ch, Co, Ev, Me, Re, Us>(message) &&
+      Array.isArray(message?.mentioned_users)
+    ) {
+      const mentionedUsers = message.mentioned_users.map((user) => user.id);
+      setMentionedUsers(mentionedUsers);
+    }
+  }, [message]);
```

Comment:
```suggestion
  }, [messageValue]);
```

Will this work as well? Generally its better to avoid objects as hook dependency

Reply:
I will have to check. Give me some time. 🤞

Reply:
@vishalnarkhede Done, it works :)